apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -42,7 +42,9 @@ import {queryParams} from '@cdo/apps/code-studio/utils';
 const HEADER_HEIGHT = styleConstants['workspace-headers-height'];
 const RESIZER_HEIGHT = styleConstants['resize-bar-width'];
 
-const MIN_HEIGHT = RESIZER_HEIGHT + 60;
+// Not sure what 60 stands for (was already there before I modified next line),
+// but +75 was added in order to prevent scroll instructions buttons to collide with borders of Instructions section
```

Comment:
I'm still concerned how this appears to have regressed since a year ago.  Adding more constants without understanding why they became necessary isn't ideal.  

If we don't have time to figure it out, at minimum we could extract the 75 into a named constant and attach a comment explaining why 75 is the right number.  Otherwise, somebody arriving at this code in a few more years will have to guess, if they are able to find this PR and screenshots.

Reply:
Good call, will do and try to explain more